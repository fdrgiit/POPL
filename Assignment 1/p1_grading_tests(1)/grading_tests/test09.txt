
  >=<===<>><>>>> > > 	>>

    12345677777777777.000012222223

  +-*/_=:;,[]().

  0101010101010 01010 00000 11111

   IF 	BEGIN	VAR	ASSIGN	DO	WHILE	PRINT	INT		REAL		STRING END	

   if var assign begin do while PrinT iNt REAl



