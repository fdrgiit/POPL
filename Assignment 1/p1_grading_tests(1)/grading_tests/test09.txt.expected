2 30 >=
2 29 <=
2 18 =
2 18 =
2 26 <>
2 27 >
2 26 <>
2 32 >>
2 27 >
2 27 >
2 27 >
2 32 >>
4 34 12345677777777777 
4 33 .
4 34 0 
4 34 0 
4 34 0 
4 34 0 
4 34 12222223 
6 13 +
6 14 -
6 17 *
6 16 /
6 15 _
6 18 =
6 19 :
6 21 ;
6 20 ,
6 22 [
6 23 ]
6 24 (
6 25 )
6 33 .
8 34 0 
8 34 101010101010 
8 34 0 
8 34 1010 
8 34 0 
8 34 0 
8 34 0 
8 34 0 
8 34 0 
8 34 11111 
10 5 IF
10 2 BEGIN
10 1 VAR
10 4 ASSIGN
10 7 DO
10 6 WHILE
10 9 PRINT
10 10 INT
10 11 REAL
10 12 STRING
10 3 END
12 37 if 
12 37 var 
12 37 assign 
12 37 begin 
12 37 do 
12 37 while 
12 37 PrinT 
12 37 iNt 
12 37 REAl 
